lib/display.ex

```diff
@@ -53,6 +53,12 @@ defmodule Display do
     format_error(error) |> IO.puts
   end
 
+  def congratulate do
+    format_green("\nYou have learned much. You must find your own path now.")
+    |> IO.puts
+    exit(:normal)
```

Comment:
Would it be possible to let something else call `exit`? It doesn't feel right for the display to manage the process lifecycle but I may be missing something.